### PR TITLE
DOCSUP-6387: Updated the toIPv6 and IPv6StringToNum functions

### PR DESCRIPTION
**Pull request by gyuton**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #19518

### CODE REVIEW

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 1]
```diff
@@ -222,43 +252,66 @@ SELECT
 └───────────────────────────────────┴──────────────────────────┘
 ```
 
-## toIPv6(string) {#toipv6string}
+## toIPv6 {#toipv6string}
+
+An alias to [IPv6StringToNum](#ipv6stringtonums) that takes a string form of IPv6 address and returns value of [IPv6](../../sql-reference/data-types/domains/ipv6.md) type, which is binary equal to value returned by `IPv6StringToNum()`.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:257]:
```suggestion
Converts a string form of IPv6 address to [IPv6](../../sql-reference/data-types/domains/ipv6.md) type. 
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 2]
```diff
@@ -222,43 +252,66 @@ SELECT
...
+
+**Parameters**
+
+-   `string` — IP address. [String](../../sql-reference/data-types/string.md)
+
+**Returned value**
 
-An alias to `IPv6StringToNum()` that takes a string form of IPv6 address and returns value of [IPv6](../../sql-reference/data-types/domains/ipv6.md) type, which is binary equal to value returned by `IPv6StringToNum()`.
-If the IP address is a valid IPv4 address then the IPv6 equivalent of the IPv4 address is returned.
+-   Converted IP address. If IPv4 passed, returns its representation as IPv6.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:273]:
```suggestion
-   IP address. 
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 3]
```diff
@@ -222,43 +252,66 @@ SELECT
 └───────────────────────────────────┴──────────────────────────┘
 ```
 
-## toIPv6(string) {#toipv6string}
+## toIPv6 {#toipv6string}
+
+An alias to [IPv6StringToNum](#ipv6stringtonums) that takes a string form of IPv6 address and returns value of [IPv6](../../sql-reference/data-types/domains/ipv6.md) type, which is binary equal to value returned by `IPv6StringToNum()`.
+
+If the IP address is a valid IPv4 address, then the IPv6 equivalent of the IPv4 address is returned.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:259]:
```suggestion
If the input string contains a valid IPv4 address, then the IPv6 equivalent of the IPv4 address is returned.
```

File: docs/ru/sql-reference/functions/ip-address-functions.md [hunk 4]
```diff
@@ -221,38 +262,62 @@ SELECT
 └───────────────────────────────────┴──────────────────────────┘
 ```
 
-## toIPv6(string) {#toipv6string}
+## toIPv6 {#toipv6string}
 
-Псевдоним функции `IPv6StringToNum()` которая принимает строку с адресом IPv6 и возвращает значение типа [IPv6](../../sql-reference/functions/ip-address-functions.md), которое равно значению, возвращаемому функцией `IPv6StringToNum()`.
+Приводит строку с адресом IPv6 к типу [IPv6](../../sql-reference/data-types/domains/ipv6.md). Возвращает пустое значение, если `string` не является корректным IP адресом.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/ip-address-functions.md:267]:
```suggestion
Приводит строку с адресом в формате IPv6 к типу [IPv6](../../sql-reference/data-types/domains/ipv6.md). Возвращает пустое значение, если аргумент не является корректным IP адресом.
```

File: docs/ru/sql-reference/functions/ip-address-functions.md [hunk 5]
```diff
@@ -221,38 +262,62 @@ SELECT
...
-┌─toTypeName(IPv6StringToNum(IPv6_string))─┬─toTypeName(toIPv6(IPv6_string))─┐
-│ FixedString(16)                          │ IPv6                            │
-└──────────────────────────────────────────┴─────────────────────────────────┘
+**Синтаксис**
+
+```sql
+toIPv6(string)
 ```
 
+**Параметры**
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/ip-address-functions.md:278]:
```suggestion
**Аргумент**
```

File: docs/ru/sql-reference/functions/ip-address-functions.md [hunk 6]
```diff
@@ -221,38 +262,62 @@ SELECT
...
+Похоже на функцию [IPv6StringToNum](#ipv6stringtonums), которая представляет адрес IPv6 в двоичном виде.
 
-``` sql
-WITH
-    '2001:438:ffff::407d:1bc1' as IPv6_string
-SELECT
-    toTypeName(IPv6StringToNum(IPv6_string)),
-    toTypeName(toIPv6(IPv6_string))
-```
+Если IP адрес является корректным IPv4 адресом, функция возвращает его IPv6 эквивалент.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/ip-address-functions.md:270]:
```suggestion
Если входящая строка содержит корректный IPv4 адрес, функция возвращает его IPv6 эквивалент.
```

File: docs/ru/sql-reference/functions/ip-address-functions.md [hunk 7]
```diff
@@ -121,12 +121,53 @@ LIMIT 10
...
-Alias: `INET6_ATON`.
+Синоним: `INET6_ATON`.
+
+**Синтаксис**
+
+``` sql
+IPv6StringToNum(string)
+```
+
+**Параметры**
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/ip-address-functions.md:140]:
```suggestion
**Аргумент** 
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 8]
```diff
@@ -232,43 +261,56 @@ SELECT
...
-┌─toTypeName(IPv6StringToNum(IPv6_string))─┬─toTypeName(toIPv6(IPv6_string))─┐
-│ FixedString(16)                          │ IPv6                            │
-└──────────────────────────────────────────┴─────────────────────────────────┘
+**Syntax**
+
+```sql
+toIPv6(string)
 ```
 
+**Parameters**
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:277]:
```suggestion
**Argument**
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 9]
```diff
@@ -121,24 +121,53 @@ LIMIT 10
...
 Alias: `INET6_ATON`.
 
+**Syntax**
+
 ``` sql
-SELECT cutIPv6(IPv6StringToNum('127.0.0.1'), 0, 0);
+IPv6StringToNum(string)
 ```
 
+**Parameters**
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:139]:
```suggestion
**Argument** 
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 10]
```diff
@@ -121,24 +121,53 @@ LIMIT 10
...
 ```
 
-## IPv6StringToNum(s) {#ipv6stringtonums}
+## IPv6StringToNum {#ipv6stringtonums}
 
-The reverse function of IPv6NumToString. If the IPv6 address has an invalid format, it returns a string of null bytes. 
-If the IP address is a valid IPv4 address then the IPv6 equivalent of the IPv4 address is returned.
+The reverse function of [IPv6NumToString](#ipv6numtostringx). If the IPv6 address has an invalid format, it returns a string of null bytes. 
+
+If the IP address is a valid IPv4 address, returns the IPv6 equivalent of the IPv4 address.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:128]:
```suggestion
If the input string contains a valid IPv4 address, returns its IPv6 equivalent.
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 11]
```diff
@@ -232,43 +261,56 @@ SELECT
 └───────────────────────────────────┴──────────────────────────┘
 ```
 
-## toIPv6(string) {#toipv6string}
+## toIPv6 {#toipv6string}
 
-An alias to `IPv6StringToNum()` that takes a string form of IPv6 address and returns value of [IPv6](../../sql-reference/data-types/domains/ipv6.md) type, which is binary equal to value returned by `IPv6StringToNum()`.
-If the IP address is a valid IPv4 address then the IPv6 equivalent of the IPv4 address is returned.
+Converts a string form of IPv6 address to [IPv6](../../sql-reference/data-types/domains/ipv6.md) type. If the IPv6 address has an invalid format, it returns  empty value.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:266]:
```suggestion
Converts a string form of IPv6 address to [IPv6](../../sql-reference/data-types/domains/ipv6.md) type. If the IPv6 address has an invalid format, returns an empty value.
```

File: docs/ru/sql-reference/functions/ip-address-functions.md [hunk 12]
```diff
@@ -121,12 +121,53 @@ LIMIT 10
...
+IPv6StringToNum(string)
+```
+
+**Параметры** 
+
+-   `string` — IP адрес. [String](../../sql-reference/data-types/string.md).
+
+**Возвращаемое значение**
+
+-   Адрес IPv6 в двоичном представление.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/ip-address-functions.md:146]:
```suggestion
-   Адрес IPv6 в двоичном представлении.
```

**Comment by gyuton** [comment]
@vdimir Hi! Can you review this, please?

**Review by vdimir — APPROVED** [review]

**Pull request merged**